DigitalLearningSolutions.Data.Tests/TestHelpers/UserTestHelper.cs

```diff
@@ -219,5 +220,25 @@ FROM Candidates
 
             return users.Single();
         }
+
+        public static AdminRoles GetDefaultAdminRoles

```

Comment:
Could you just do new AdminRoles() instead of defining this new helper method as the constructor of AdminRoles sets everything to true by default?